polyfill/spatial-navigation-polyfill.js

```diff
@@ -367,7 +403,9 @@
   function spatialNavigationSearch (dir, args) {
     let {candidates, container, outsideOnly} = args || {};
     const targetElement = this;
-    candidates = spatNavCandidates(targetElement, dir, candidates, container, outsideOnly);
+    let bestTarget;
+    let selectBestFunction = selectBestCandidate;
```

Comment:
What is the purpose of this variable?

Reply:
It's unused variable. I removed it. thanks!